scripts/mariadb/create_stats_schema.sql

```diff
@@ -119,4 +129,192 @@ CREATE TABLE
 		Hits MEDIUMINT UNSIGNED NOT NULL,
 		PRIMARY KEY (WeaponStatsID),
 		CONSTRAINT `fk_class_stats` FOREIGN KEY (ClassStatsID) REFERENCES ClassStats (ClassStatsID) ON DELETE CASCADE ON UPDATE RESTRICT
-	) ENGINE = InnoDB;
\ No newline at end of file
+	) ENGINE = InnoDB;
+
+CREATE TABLE
+	IF NOT EXISTS OverallStatsAllTime (
+		SteamID BIGINT UNSIGNED NOT NULL,
+		PlayerName VARCHAR(32) CHARACTER,
+		NumberOfGames SMALLINT UNSIGNED,
+		AverageDPM FLOAT,
+		AverageKills FLOAT,
+		AverageAssists FLOAT,
+		AverageDeaths FLOAT,
+		AverageAirshots FLOAT,
+		AverageHeadshots FLOAT
+		SET
+			utf8mb4 COLLATE utf8mb4_unicode_ci,
+			PRIMARY KEY (SteamID)	
+	)
+
+CREATE TABLE
+	IF NOT EXISTS OverallStatsRecent (
+		SteamID BIGINT UNSIGNED NOT NULL,
+		PlayerName VARCHAR(32) CHARACTER,
+		NumberOfGames SMALLINT UNSIGNED NOT NULL,
+		AverageDPM FLOAT,
+		AverageKills FLOAT,
+		AverageAssists FLOAT,
+		AverageDeaths FLOAT,
+		AverageAirshots FLOAT,
+		AverageHeadshots FLOAT
+		SET
+			utf8mb4 COLLATE utf8mb4_unicode_ci,
+			PRIMARY KEY (SteamID)	
+	)
+
+CREATE TABLE
+	IF NOT EXISTS ScoutRecent (
+		SteamID BIGINT UNSIGNED NOT NULL,
+		PlayerName VARCHAR(32) CHARACTER,
+		NumberOfGames SMALLINT UNSIGNED,
+		AverageDPM FLOAT,
+		AverageKills FLOAT,
+		AverageAssists FLOAT,
+		AverageDeaths FLOAT
+		SET
+			utf8mb4 COLLATE utf8mb4_unicode_ci,
+			PRIMARY KEY (SteamID)	
+	)
+
+CREATE TABLE
+	IF NOT EXISTS ScoutAllTime (
+		SteamID BIGINT UNSIGNED NOT NULL,
+		PlayerName VARCHAR(32) CHARACTER,
```

Comment:
can you use 255 for the size here?
I had issues with some long names of stuff that screwed stuff up